### PR TITLE
Elastic v3

### PR DESCRIPTION
**Pull request by i-like-robots**
- [x] Display title, author, main image and content
- [x] Display tags and primary tag
- [x] Display more ons
- [ ] Display suggested reads
- [x] Integrate DFP
- [x] Integrate barriers and first click free
- [x] Integrate Open Graph
- [x] Integrate Twitter card

It's probably OK to merge this without reaching full feature parity as it's behind a flag just so the PR doesn't get ridiculous.

After we drop support the V1 and V2 indexes this controller can be refactored somewhat, a lot of the mapping about will be redundant.

### CODE REVIEW

**Comment by matthew-andrews** [comment]
:+1:

File: server/controllers/negotiation.js [hunk 1]
```diff
@@ -40,6 +47,7 @@ function getArticleV1(contentId, flags) {
 function getArticleV2(contentId, flags) {
 	return api.content({
 		uuid: contentId,
+		inex: 'v2_api_v2',
```
Comment (andygnewman) [inline server/controllers/negotiation.js]: was this meant to be `index: `

**Comment by matthew-andrews** [comment]
:boom:

**Comment by andygnewman** [comment]
:8ball:

**Pull request merged**